extensions/pkg/controller/cmd/options.go

```diff
@@ -425,12 +429,24 @@ func (d *SwitchOptions) AddFlags(fs *pflag.FlagSet) {
 	for name := range d.nameToAddToManager {
 		controllerNames = append(controllerNames, name)
 	}
+	fs.StringSliceVar(&d.Enabled, ControllersFlag, controllerNames, fmt.Sprintf("List of controllers to enable %v", controllerNames))
 	fs.StringSliceVar(&d.Disabled, DisableFlag, d.Disabled, fmt.Sprintf("List of controllers to disable %v", controllerNames))
```

Comment:
I might have figured out the idea behind having an `enabled` and a `disabled` flag 😄 

In the `controllerregistration` case all controllers are enabled by default. Depending on the use case, some user's might disable some controllers (like [here](https://github.com/gardener/gardener/pull/10518/files#diff-c70d5320477a9d09e2d2ae1434479cb8560d55dad1d611fada9345b5537a3390L71) in the helm chart). In this case it is more convenient to disable just a these few controllers instead of enable every other controller.

In the `deploy via operator` case only [backupbucket and dns controller](https://github.com/gardener/gardener/pull/10518/files#diff-c70d5320477a9d09e2d2ae1434479cb8560d55dad1d611fada9345b5537a3390R73) are required. In this case it is more convenient to enable only these controllers instead of disabling every other controller.